examples/dao/assets/deposit-lsig.py

```diff
@@ -0,0 +1,72 @@
+import sys
+sys.path.insert(0,'..')
+
+from algobpy.parse import parse_params
+from pyteal import *
+
+def deposit_lsig(ARG_GOV_TOKEN, ARG_DAO_APP_ID):
+    """
+    A logic signature account which holds:
+    - vote token deposits
+    - Proposal deposits
+    """
+
+    # check no rekeying, close remainder to, asset close to for a txn
+    def basic_checks(txn: Txn): return And(
+        txn.rekey_to() == Global.zero_address(),
+        txn.close_remainder_to() == Global.zero_address(),
+        txn.asset_close_to() == Global.zero_address()
+    )
+
+    # Opt-in transaction. We only accept gov-token in this contract
+    opt_in = And(
+        basic_checks(Txn),
+        Txn.type_enum() == TxnType.AssetTransfer,
+        Txn.asset_amount() == Int(0),
+        Txn.xfer_asset() == Int(ARG_GOV_TOKEN)
+    )
+
+    # Verifies transaction group in case of deposting votes or withdrawing them
+    def deposit_or_withdraw(app_arg_1: Bytes, app_arg_2: Bytes): return And(
+        Global.group_size() == Int(2),
+
+        # verify first transaction
+        basic_checks(Gtxn[0]),
+        Gtxn[0].type_enum() == TxnType.ApplicationCall,
+        Gtxn[0].application_id() == Int(ARG_DAO_APP_ID),
+        Or(
+            Gtxn[0].application_args[0] == app_arg_1,
+            Gtxn[0].application_args[0] == app_arg_2
```

Comment:
Can we instead of `payment = Or(....`, can we have a bigger `Or` statement here, where we check if : 
```
Gtxn[0].application_args[0] is in {"add_proposal", "deposit_vote_token", "withdraw_vote_deposit", "clear_proposal")
```

Reply:
yes, looks simpler.